### DISCUSSION TITLE
Sent Create of Endless Resist storage to the wrong character

### DISCUSSION THREAD

**Discussion by B4nanaboi** [chromiecraft/chromiecraft#5035] [General]
I'm sorry to post this in discussions here, but I don't have Discord acces. Maybe you could easily fix this and delete this post afterwards?

I just bought a Crate of Endless resist storage in the shop yet sent it to the wrong character (see details below). Is there a way to send this bag to Bananaboi instead (same account) and remove it from Skippert? Both chars are from the same account. I would greatly appreciate it for the 50-60 bucks :( Thanks in advance.

![image](https://user-images.githubusercontent.com/121559482/218587218-0b6555d4-032f-4b7e-90e7-df7d57a24472.png)

**Comment by Gultask** [comment]
Hey @B4nanaboi could you click the 'Return' button in the mail?
We'll take care of this soon, either today or tomorrow.

**Comment by B4nanaboi** [comment]
Hej @Gultask , thanks for your quick reply. This tuesday evening I'll click the return button in the email. (no acces at work right now). I'll post here when I've returned the email. Cheers

**Comment by B4nanaboi** [comment]
@Gultask  I've just returned the email that I'd received at Skippert. Is it possible to send the item to Bananaboi? Thank you very much in advance.

**Reply by Gultask** [reply]
Let me know if it arrives.

**Reply by B4nanaboi** [reply]
It's resolved. Feel free to close/delete the thread if you want. Cheers! Thank you very much for the service.
